code/modules/hydroponics/grown/nettle.dm

```diff
@@ -105,13 +105,6 @@
 /obj/item/grown/nettle/death/attack(mob/living/carbon/M, mob/user)
 	..()
 	if(isliving(M))
-		to_chat(M, "<span class='danger'>You are stunned by the powerful acid of the Deathnettle!</span>")
+		to_chat(M, "<span class='danger'>You flinch as you are struck by the Deathnettle!</span>")
 		add_attack_logs(user, M, "Hit with [src]")
-
-		M.AdjustEyeBlurry((force / 7) STATUS_EFFECT_CONSTANT)
-		if(prob(20))
-			var/paralyze_time = (force * 10 / 3) SECONDS
-			var/stun_time = (force / 7.5) SECONDS
-			M.Paralyse(paralyze_time)
-			M.Weaken(stun_time)
-		M.drop_item()
+		M.AdjustEyeBlurry(force * 2) // Maximum duration 5 seconds.
```

Comment:
```suggestion
		M.AdjustEyeBlurry((force / 5) SECONDS) // Maximum duration 5 seconds per hit.
```

Note that shis stacks infinitely on successive applications. Although with [up to] 25 damage a hit, eye blur would be the least of your problems.

Reply:
> Note that shis stacks infinitely on successive applications. Although with [up to] 25 damage a hit, eye blur would be the least of your problems.

Eye blur stacking was preferred after speaking with balance, just decided to specify it stacked in the comment.

Reply:
Sorry, i kinda jammed two unrelated things in the same comment, confusing you.
Let me try this again

1) I'm fine with stacking eyeblur; thanks for confirmation that this is intentded.

2) `(force / 5) SECONDS` and `force * 2` do exact same thing, but the former is easier to read, since you don't need to remember that `.AdjustX` procs operate in ticks rather than seconds. So i suggest changing it to `(force / 5) SECONDS`